File: WeekendTask/python/perfect_number.py
#perfect number
number= 6
add = 0
for count in range(number):
	if(count % 2 == 0):
		add +=count

#	if sum is equal to the number used for division
	
if (add == number):
	print("It's a perfect number")
	
else:
	print("It is not a perfect number")

File: WeekendTask/javascript/PerfectNumber.js
//perfect number

		let number= 6;
		let sum = 0;
		for(let count=0; count<number; count++){
	if(count % 2 == 0){
		sum +=count;
	
	}
	}
	//is sum is equal to the number used for division
	
	if (sum == number){
	console.log("It's a perfect number");
	}
else{
console.log("It is not a perfect number");
}
	



